### PR TITLE
chore: improve docs

### PR DESCRIPTION
**Pull request by waterplea**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes # 

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo/src/pages/documentation/mask-expression/mask-expression.template.html [hunk 1]
```diff
@@ -98,55 +103,72 @@ <h2>Types of mask expression</h2>
...
+                    class="tui-space_bottom-3"
+                >
+                    An "element state" object with raw value and current
+                    selection is passed as an argument to the function. See more
+                    about element state in
+                    <a
+                        tuiLink
+                        [routerLink]="processorsDocPage"
+                    >
+                        Processors
```
Comment (nsbarsukov) [inline projects/demo/src/pages/documentation/mask-expression/mask-expression.template.html:149]: Probably, we should create the separate topic "Element state" inside "Core concepts" 🤔 

I will create issue for it now

Comment (nsbarsukov) [reply projects/demo/src/pages/documentation/mask-expression/mask-expression.template.html:149]: - https://github.com/Tinkoff/maskito/issues/152

File: projects/demo/src/pages/kit/date-range/date-range-mask-doc.template.html [hunk 2]
```diff
@@ -35,8 +35,8 @@
                 <code>min</code>
                 and
                 <code>max</code>
-                allow to set the least and the greatest available dates. They
-                accept native
+                allow you to set the earliest and the latest available dates.
```
Comment (nsbarsukov) [inline projects/demo/src/pages/kit/date-range/date-range-mask-doc.template.html:38]: Should we change API-page too (`min` and `max` descriptions) ?

File: projects/demo/src/pages/kit/date/date-mask-doc.template.html [hunk 3]
```diff
@@ -72,8 +72,8 @@
                 <code>min</code>
                 and
                 <code>max</code>
-                allow to set the least and the greatest available dates. They
-                accept native
+                allow you to set the earliest and the latest available dates.
```
Comment (nsbarsukov) [inline projects/demo/src/pages/kit/date/date-mask-doc.template.html:75]: API page

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**